src/uu/cksum/src/cksum.rs

```diff
@@ -186,17 +194,33 @@ where
             ));
             continue;
         }
-        if options.raw {
-            let bytes = match options.algo_name {
-                ALGORITHM_OPTIONS_CRC => sum.parse::<u32>().unwrap().to_be_bytes().to_vec(),
-                ALGORITHM_OPTIONS_SYSV | ALGORITHM_OPTIONS_BSD => {
-                    sum.parse::<u16>().unwrap().to_be_bytes().to_vec()
+        let sum = match options.output_format {
+            OutputFormat::Raw => {
+                let bytes = match options.algo_name {
+                    ALGORITHM_OPTIONS_CRC => sum_hex.parse::<u32>().unwrap().to_be_bytes().to_vec(),
+                    ALGORITHM_OPTIONS_SYSV | ALGORITHM_OPTIONS_BSD => {
+                        sum_hex.parse::<u16>().unwrap().to_be_bytes().to_vec()
+                    }
+                    _ => decode(sum_hex).unwrap(),
+                };
+                // Cannot handle multiple files anyway, output immediately.
+                stdout().write_all(&bytes)?;
+                return Ok(());
+            }
+            OutputFormat::Hexadecimal => sum_hex,
+            OutputFormat::Base64 => {
+                match options.algo_name {
+                    ALGORITHM_OPTIONS_CRC | ALGORITHM_OPTIONS_SYSV | ALGORITHM_OPTIONS_BSD => {
+                        sum_hex
+                    }
+                    _ => {
+                        //data_encoding::BASE64.encode(decode(sum_hex).unwrap())
```

Comment:
could you please remove this comment ?

Reply:
Oh! Sorry, I must have been more tired than I thought. Thank you!